crates/common/consensus/src/misc.rs

```diff
@@ -10,3 +10,131 @@ pub fn compute_signing_root<SSZObject: TreeHash>(ssz_object: SSZObject, domain:
     }
     .tree_hash_root()
 }
+use std::cmp::max;
+
+use alloy_primitives::aliases::B32;
+use anyhow::ensure;
+use ethereum_hashing::hash;
+
+use crate::{
+    deneb::beacon_state::BeaconState,
+    fork_choice::helpers::constants::{
+        MAX_EFFECTIVE_BALANCE, MAX_SEED_LOOKAHEAD, SHUFFLE_ROUND_COUNT, SLOTS_PER_EPOCH,
+    },
+    fork_data::ForkData,
+};
+
+static MAX_RANDOM_BYTE: u64 = 255;
```

Comment:
```suggestion
const MAX_RANDOM_BYTE: u64 = 255;
```
shouldn't this be a constant as well